client/lib/plans/constants.js

```diff
@@ -561,7 +561,7 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'jetpack-personal',
 		getDescription: () =>
 			i18n.translate(
-				'Security essentials for your WordPress site including ' +
+				'Security essentials for your WordPress site, including ' +
```

Comment:
There is a second occurrence of this string, which lacks the edits.

Reply:
Thanks for the catch. Got it.